src/engraving/dom/chord.cpp

```diff
@@ -2946,7 +2963,16 @@ EngravingItem* Chord::prevElement()
             return m_arpeggio;
         }
     // fall through
-
+    case ElementType::TREMOLO_TWOCHORD:
+        if (m_arpeggio) {
+            return m_arpeggio;
+        }
+    // fall through
+    case ElementType::TREMOLO_SINGLECHORD:
```

Comment:
Although it doesn't seem to produce incorrect results, it seems that the relation with the `// fall through` is not entirely as intended now, because now, in case of `TREMOLO`, it checks `m_arpeggio` three times (at least if it's `nullptr`).

Maybe better do
```c++
    case ElementType::TREMOLO:
    case ElementType::TREMOLO_SINGLECHORD:
    case ElementType::TREMOLO_TWOCHORD:
        if (m_arpeggio) {
            return m_arpeggio;
        }
    // fall through
```